crates/ika-sui-client/src/lib.rs

```diff
@@ -294,6 +295,17 @@ where
         }
     }
 
+    pub async fn get_clock(&self) -> IkaResult<Clock> {
+        let result = self
+            .inner
+            .get_clock(ObjectID::from_hex_literal("0x6").unwrap())
```

Comment:
```suggestion
            .get_clock(ObjectID::from_hex_literal("0x6").unwrap())
```
let clock_address = "0x6";